src/iris/ui/__init__.py

```diff
@@ -69,42 +72,44 @@ def get_local(req, path):
     return requests.get('http://127.0.0.1:16649/v0/%s' % path, cookies=req.cookies).json()
 
 
-def send_file(req, resp):
-    if not req.path.startswith('/static'):
-        raise HTTPNotFound()
-    filepath = os.path.join(ui_root, req.path[1:])
-    try:
-        resp.stream = open(filepath, 'rb')
-        resp.stream_len = os.path.getsize(filepath)
-    except IOError:
-        raise HTTPNotFound()
-
-
-def static_assets(req, resp):
-    if req.path.endswith('.js'):
-        resp.content_type = 'text/javascript'
-    elif req.path.endswith('.css'):
-        resp.content_type = 'text/css'
-    send_file(req, resp)
-
+# Credit to Werkzeug for implementation
+def secure_filename(filename):
+    for sep in os.path.sep, os.path.altsep:
+        if sep:
+            filename = filename.replace(sep, ' ')
+    filename = str(_filename_ascii_strip_re.sub('', '_'.join(
+        filename.split()))).strip('._')
+    return filename
 
-def static_imgs(req, resp):
-    if req.path.endswith('.png'):
-        resp.content_type = 'image/png'
-    elif req.path.endswith('.jpg'):
-        resp.content_type = 'image/jpg'
-    elif req.path.endswith('.svg'):
-        resp.content_type = 'image/svg+xml'
-    send_file(req, resp)
 
+class StaticResource(object):
+    allow_read_only = False
```

Comment:
I think `allow_read_only` should be set to True and `frontend_route` should be set to False, so these static assets are available to the login form.